config/routes.rb

```diff
@@ -62,7 +62,9 @@
       get 'simulations/new_exemplar', to: 'simulations#new_exemplar', as: :new_exemplar_simulation
       resources :simulations
 
-      resources :alerts#, only: [:show, :index]
+      resources :alerts, only: [:index, :show] do
+        resource :find_out_more, controller: :find_out_more
```

Comment:
Separate controller action for find out more. I think we'll have a more complex structure and I don't think we're just "showing" an alert here.